packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -97,7 +100,9 @@ const calculateSizeParameters = (
   const currentMaxSize = getCurrentMaxSize(columnModel)
   const recommendedSize = getRecommendedMaxSize(defaultAnnotationModel)
   const isSmallerThanRecommendedSize =
-    recommendedSize && currentMaxSize ? currentMaxSize < recommendedSize : false
+    recommendedSize && currentMaxSize && currentMaxSize > 0
+      ? currentMaxSize < recommendedSize
+      : false
```

Comment:
Eliminate case of overlap between errorsByField and isSmallerThanRecommendedSize when input is a negative number